src/main/views/template.njk

```diff
@@ -1,38 +1,53 @@
+          </svg>
+          <!--<![endif]-->
+          <!--[if IE 8]>
+          <img src="/assets/images/govuk-logotype-crown.png" class="govuk-header__logotype-crown-fallback-image"
+               width="36" height="32">
+          <![endif]-->
+          <span class="govuk-header__logotype-text">
+            GOV.UK
+          </span>
+        </span>
+        </a>
+      </div>
+      <div class="govuk-header__content">
+        <a href="#" class="govuk-header__link govuk-header__link--service-name">
+          {{ serviceNameText }}
+        </a>
+        <div class="govuk-form-group header-select">
+          <select class="govuk-select" id="select-option" name="select option" aria-label="Dropdown navigation" onchange="location = this.value;">
+            <option disabled selected value=""></option>
+            <option value="otp-login">My subscriptions</option>
+            <option value="single-justice-procedure-search">View Public Single Justice Procedure</option>
+            <option value="live-case-alphabet-search">Find Live Case Status Updates</option>
+            <option value="">Find a publication</option>
```

Comment:
missing value for here, wouldnt it be the `alphabet-search`? or the how do you wanna search page?

Reply:
Yeah good point I will add that thanks.